### ISSUE TITLE
Rename building=train_station preset

### ISSUE THREAD

**Issue by lakedistrictOSM** [openstreetmap/iD#5903]
https://wiki.openstreetmap.org/wiki/Tag:building=train%20station

Currently the building=train_station preset is named "Train Station", which is the same name as the railway=station preset.

The tag is supposed to be used for train station buildings, and should not be discouraged as suggested in #3779.

A station can have multiple station buildings, but should have only one node or surrounding area tagged railway=station.

Please consider renaming the building=train_station preset "Train Station Building" or "Station Building" and make the preset searchable once more.

**Comment by quincylvania** [comment]
I renamed it to Train Station Building since it is confusing for them both to have the same name.

I'm not sure about making it searchable. I for one didn't know the difference between building features and building-like POIs when I first started mapping. If I was a new mapper who searched "train station" and saw both "Train Station" and "Train Station Building", I might think "this train station is also a building, so I'll pick that one". Then again, we have a few similar situations with the other behavior. Like "Stadium Building" (`building=stadium`) is searchable and distinct from "Stadium" (`leisure=stadium`). I'll leave this open for further discussion.
